server/market/market.go

```diff
@@ -106,18 +106,18 @@ type Config struct {
 // including order status, history, cancellation statistics, etc.
 //
 // The Market performs the following:
-// - Receiving and validating new order data (amounts vs. lot size, check fees,
-//   utxos, sufficient market buy buffer, etc.).
-// - Putting incoming orders into the current epoch queue.
-// - Maintain an order book, which must also implement matcher.Booker.
-// - Initiate order matching via matcher.Match(book, currentQueue)
-// - During and/or after matching:
+//  1. Receiving and validating new order data (amounts vs. lot size, check
+//     fees, utxos, sufficient market buy buffer, etc.).
+//  2. Putting incoming orders into the current epoch queue.
+//  3. Maintain an order book, which must also implement matcher.Booker.
+//  4. Initiate order matching via matcher.Match(book, currentQueue)
+//  5. During and/or after matching:
 //     * update the book (remove orders, add new standing orders, etc.)
 //     * retire/archive the epoch queue
 //     * publish the matches (and order book changes?)
 //     * initiate swaps for each match (possibly groups of related matches)
-// - Cycle the epochs.
-// - Recording all events with the archivist
+//  6. Cycle the epochs.
+//  7. Recording all events with the archivist
```

Comment:
Slightly unrelated to the purpose of the pr, but except for `#5` maybe should be present tense and lose the -ing for the action verb.